webhooks-extension/docs/InstallReleaseBuild.md

```diff
@@ -0,0 +1,69 @@
+# Installing Official Releases
+<br/>
+
+Run the [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply) command to perform installation
+<br/>
+<br/>
+
+  * **To install latest:**
+
+    _On Red Hat OpenShift:_
+
+    ```bash
+    kubectl apply --filename https://github.com/tektoncd/dashboard/releases/latest/download/openshift-webhooks-extension.yaml
+    ```
+
+    _On other clouds:_
```

Comment:
For other platforms*? My Mac for example isn't a cloud 😉

Reply:
changed to "kubernetes environments"